README.md

```diff
@@ -263,6 +257,119 @@ This would create a structure such as:
 
 Note the special string `COL_VAL` - this matches the CSV column value for the row being processed.
 
+## create-users
+
+
+## Creating a new user with a new place.
+
+To create new users associated to new place and a new contact. Provide values for contact.name, place.name, and place.parent(can be existing place)
+
+```
+username,password,roles,name,phone,contact.name,place.c_prop,place.type,place.name,place.parent
+alice,Secret_1,district-admin,Alice Example,+123456789,Alice,p_val_a,health_center,alice area, district_uuid
+bob,Secret_1,district-admin,bob Example,+123456789,bob,p_val_a,health_center,bob area,disctrict_uuid
+```
+
+### Linking users to contacts created from csv-to-docs
+
+To create user accounts for contacts that are created while running csv-to-docs action follow these steps.
+
+1. Create a `users.csv` file in the `csv` folder with the rest of the csvs needed for `csv-to-docs` action.
+1. Add columns for username, password, roles, phone, contact, place, and any other additional fields you want to populate. 
+1. Using the query language for contact `contact:person WHERE reference_id=COL_VAL` and place `place:GET _id OF place WHERE reference_id=COL_VAL` to fill these values based on the generated UUID for contact and place created by `csv-to-docs`. This works the same as if you were using `csv-to-docs`
```

Comment:
Maybe something like: "Use the following query language as the header names: for contact `contact:person WHERE reference_id=COL_VAL` and place `place:GET _id OF place WHERE reference_id=COL_VAL`. This feature is also supported in the `csv-to-docs` csv files."